e2e/test_dataset_manager.py

```diff
@@ -59,7 +61,46 @@ def tearDown(self):
         self.dataset_manager.remove_dataset()
 
 
-class PartitionedDatasetManagerPropertiesTestCase(DatasetManagerTestCase):
+class DatasetManagerTestCase(DatasetManagerBaseTestCase):
+    def test_should_have_labels(self):
```

Comment:
I don't understand this test, can you explain?

Reply:
It tests if after creating dataset this dataset has labels

Reply:
then call it `test_should_add_labels`